### ISSUE TITLE
Add support for a input list.

### ISSUE THREAD

**Issue by whizzzkid** [yask123/Instant-Music-Downloader#7]
How about supporting a file containing search terms and downloading concurrently.

**Comment by ScoreUnder** [comment]
Sounds like a job for gnu parallel
```
parallel --tag 'printf %s\\n {} | python2 music_downloader.py' < searchterms
```

**Comment by yask123** [comment]
Maybe just ask users to enter keywords  for separate songs separated by "," delimiter ?
```
>> hey jude Beatles, let it be Beatles , .... 
Downloading hey jude Beatles
....
Downloading  let it be Beatles
....
```

And if someone wants to input from a file, they can simply redirect `stdin` from a file containing the keywords
Eg:
```
$ cat text.txt
I wanna hold your hand, ...

$ python music_downloader.py < test.txt
Downloading The Beatles - I Want To Hold your Hand
```

**Comment by mondayrain** [comment]
Just made a pull request so we can accept songs, lists of songs, and files from the command line: #24. @whizzzkid, was that what you were looking for?

**Comment by yask123** [comment]
Fixed in b26c88d9f6cd636f8384265d025828be652a27fd
